bnd-workspace/de.fraunhofer.abm.app/static/de.fraunhofer.abm/template/createCollection.htm

```diff
@@ -1,13 +1,20 @@
+<head>
+<style>
```

Comment:
It is bad style to have the style tags in the htm files. Use the css files the application already has.